cmd/iacinventorycmd/iacinventorycmd.go

```diff
@@ -8,17 +8,27 @@ import (
 
 func Command() *cobra.Command {
 	tool := &iacinventory.GithubIacInventoryScanner{}
+	var directories []string
 	opts := tools.ToolOpts{}
 	c := &cobra.Command{
 		Use:   "iac-inventory",
 		Short: "Look for infrastructure-as-code and optionally send the inventory to Soluble",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return opts.RunTool(tool)
+			if directories == nil {
```

Comment:
I think this would be better as 2 different subcommands:

iac-inventory local
iac-inventory github